var/spack/repos/builtin/packages/gchp/package.py

```diff
@@ -15,15 +15,17 @@ class Gchp(CMakePackage):
 
     maintainers = ['williamdowns']
 
+    version('13.0.2', git='https://github.com/geoschem/GCHP.git',
+            commit='017ad7276a801ab7b3d6945ad24602eb9927cf01',  submodules=True)
     version('13.0.1', git='https://github.com/geoschem/GCHP.git',
             commit='f40a2476fda901eacf78c0972fdb6c20e5a06700',  submodules=True)
     version('13.0.0', git='https://github.com/geoschem/GCHP.git',
             commit='1f5a5c5630c5d066ff8306cbb8b83e267ca7c265',  submodules=True)
+    version('dev', branch='dev', submodules=True)
```

Comment:
This probably needs a `git=` argument. Can you try to put a single attribute at the class level below `url`?

Reply:
Forgot to bring that over, thanks!